openvdb/openvdb/viewer/Viewer.cc

```diff
@@ -328,7 +328,9 @@ ThreadManager::doView()
     // This function runs in its own thread.
     // The mClose and mRedisplay flags are set from the main thread.
     while (!mClose) {
-        if (mRedisplay.compare_and_swap(/*set to*/false, /*if*/true)) {
+        bool expected = true;
+	mRedisplay.compare_exchange_strong(expected, false);
+        if (expected) {
```

Comment:
This was the one tricky part, since the argument order and semantics are different. 

The tbb version is https://www.threadingbuildingblocks.org/docs/help/tbb_userguide/Atomic_Operations.html:
* `x.compare_and_swap(y,z)`: if `x` equals `z`, then do `x=y`. In either case, return old value of `x`.
The std version https://en.cppreference.com/w/cpp/atomic/atomic/compare_exchange does
* Atomically compares the object representation (until C++20)value representation (since C++20) of `*this` with that of `expected`, and if those are bitwise-equal, replaces the former with desired (performs read-modify-write operation). Otherwise, loads the actual value stored in `*this` into `expected` (performs load operation).
* Returns `true` if the underlying atomic value was successfully changed, `false` otherwise.